datagrepper/app.py

```diff
@@ -776,3 +781,14 @@ def internal_error(error):
         status=500,
         mimetype="application/json",
     )
+
+
+def liveness():
+    pass
+
+
+def readiness():
+    try:
+        datagrepper.ping()
+    except Exception:
+        raise HealthError("Can't connect to the Datagrepper server")
```

Comment:
I don't know what the `ping()` function does but I suspect it connects to the database server, not the "Datagrepper" server.